include/znc/IRCSock.h

```diff
@@ -135,6 +136,7 @@ class CIRCSock : public CIRCSocket {
 	bool                                m_bAwayNotify;
 	bool                                m_bAccountNotify;
 	bool                                m_bExtendedJoin;
+	bool                                m_bServerTime;
```

Comment:
Storing the boolean should be practically free due to memory alignment and padding, and potentially useful if we introduce a ShowNetwork -like command that shows all the details of a network.